addons/web/controllers/webclient.py

```diff
@@ -79,16 +79,15 @@ def translations(self, unique, mods=None, lang=None):
 
         translations_per_module, lang_params = request.env["ir.http"].get_translations_for_webclient(mods, lang)
 
-        body = json.dumps({
+        body = {
             'lang': lang,
             'lang_parameters': lang_params,
             'modules': translations_per_module,
             'multi_lang': len(request.env['res.lang'].sudo().get_installed()) > 1,
-        })
-        response = request.make_response(body, [
+        }
+        response = request.make_json_response(body, [
             # this method must specify a content-type application/json instead of using the default text/html set because
             # the type of the route is set to HTTP, but the rpc is made with a get and expects JSON
```

Comment:
Comment is made irrelevant, and the json changes at large don't seem mentioned in the commit message